docs/DeveloperGuide.md

```diff
@@ -31,7 +31,7 @@ title: Developer Guide
 
 * {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 * Adapted from [_SE-Education_](https://se-education.org/addressbook-level3/DeveloperGuide.html) 's original *AddressBook*
-
+* Test cases from UI adapted from [SE-Education AddressBook-Level 4](https://github.com/se-edu/addressbook-level4/tree/master/src/test/java/guitests)
```

Comment:
I think the latest version of the DG has included a similar line already, so just take note when merging.

Reply:
Ditto! If there is conflict pls keep the master!